lib/src/test/java/space/taran/arklib/ResourceIdTest.kt

```diff
@@ -0,0 +1,21 @@
+package space.taran.arklib
+
+import dev.arkbuilders.arklib.computeId
+import junit.framework.TestCase.assertEquals
+import org.junit.Test
+import kotlin.io.path.createTempFile
+import kotlin.io.path.fileSize
+import kotlin.io.path.writeText
+
+class ResourceIdTest {
+    @Test
+    fun resourceIdTest() {
+        System.loadLibrary("arklib")
+        val file = createTempFile()
+        file.writeText("hello")
+        assertEquals(
+            computeId(file.fileSize(), file).toString(),
+            "5-907060870"
+        )
+    }
```

Comment:
Let's move `System.loadLibrary` into `@Before` function and create one more small test case with computing id directly from bytes.

Reply:
We have binding only for files